RULES.md

```diff
@@ -451,8 +453,29 @@ Trip frequencies must not overlap in time
 ##### References:
 * [frequencies.txt specification](http://gtfs.org/reference/static/#frequenciestxt)
 
+<a name="PathwayToPlatformWithBoardingAreasNotice"/>
+
+#### PathwayToPlatformWithBoardingAreasNotice
+
+A pathway has an endpoint that is a platform which has boarding areas. A platform that has boarding
```

Comment:
Here, @isabelle-dr @MobilityData/transit-specs do you think that it's worth updating the specification to clarify this rule (make it more explicit)?